android/BOINC/app/src/main/java/edu/berkeley/boinc/adapter/PrefsListAdapter.java

```diff
@@ -106,6 +108,9 @@ public View getView(int position, View convertView, ViewGroup parent) {
 	    				case GIGABYTES:
 	    					value = Formatter.formatShortFileSize(this.activity, (long)(item.status.doubleValue() * 0x40000000));
 	    					break;
+						case DAYS:
```

Comment:
Please fix indents here.
BTW, maybe it is better to add `UNIT` type `DECIMAL `as more generic than `DAYS`?

Reply:
You're right, changed and fixed.